scripts/time-filter.js

```diff
@@ -0,0 +1,116 @@
+import { textSummary } from 'https://jslib.k6.io/k6-summary/0.0.3/index.js';
+import http from 'k6/http';
+import {
+    check,
+    sleep
+} from 'k6';
+import {
+    generateRandomCoordinate,
+    destinations,
+    timeFilterOptions,
+    setThresholdsForScenarios,
+    countries
+} from './common.js';
+
+export let options = timeFilterOptions
+
+setThresholdsForScenarios(options)
+
+export default function() {
+    const appId = __ENV.APP_ID;
+    const apiKey = __ENV.API_KEY;
+    const host = __ENV.HOST || 'api-dev.traveltimeapp.com';
+    const country = __ENV.COUNTRY || 'GB'
+    const countryCode = country.toUpperCase()
+    const countryCoords = countries[countryCode]
+    const url = `https://${host}/v4/time-filter`;
+    const transportation = __ENV.TRANSPORTATION || 'driving+ferry'
+    const travelTime = __ENV.TRAVEL_TIME || 1900
+    const destinationsAmount = __ENV.SCENARIO_DESTINATIONS || 2000
+    const rangeWidth = __ENV.RANGE || 0
+    const rangeSettings = {
+        enabled: rangeWidth === 0 ? false : true,
+        max_results: 3,
+        width: rangeWidth === 0 ? 1 : parseInt(rangeWidth)
+    };
+
+    const params = {
+        headers: {
+            'Content-Type': 'application/json',
+            'X-Application-Id': appId,
+            'X-Api-Key': apiKey
+        },
+    };
+
+    const response = http.post(url, generateBody(countryCode, travelTime,
+        transportation, destinationsAmount, rangeSettings, countryCoords), params);
+    check(response, {
+        'status is 200': (r) => r.status === 200,
+        'response body is not empty': (r) => r.body.length > 0,
+    });
+    sleep(1);
+}
+
+export function handleSummary(data) {
+    // removing default metrics
+    delete data.metrics['http_req_duration']
+    delete data.metrics['http_req_sending']
+    delete data.metrics['http_req_receiving']
+    delete data.metrics['http_req_blocked']
+    delete data.metrics[`http_req_duration{expected_response:true}`]
+    delete data.metrics['http_req_waiting']
+    delete data.metrics['http_reqs']
+    delete data.metrics['iteration_duration']
+    delete data.metrics['iterations']
+    delete data.metrics['vus']
+    delete data.metrics['http_req_connecting']
+    delete data.metrics['http_req_failed']
+    delete data.metrics['http_req_tls_handshaking']
+
+    data = destinations.reduce((curData, curDestinations) => {
+        return reportPerDestination(curData, curDestinations)
+    }, data)
+
+    return {
+        stdout: textSummary(data, { indent: ' ', enableColors: true }),
+    }
+}
+
+function reportPerDestination(data, destinations) {
+    data.metrics[`http_req_sending(${destinations} destinations)`] = data.metrics[`http_req_sending{scenario:sending_${destinations}_destinations}`]
```

Comment:
we'll add linter later but the maximum line length shouldn't exceed 80 characters

Reply:
linted code manually